distribution/pom.xml

```diff
@@ -448,8 +448,6 @@
                                         <argument>org.apache.druid.extensions.contrib:aliyun-oss-extensions</argument>
                                         <argument>-c</argument>
                                         <argument>org.apache.druid.extensions.contrib:opentelemetry-emitter</argument>
-                                        <argument>-c</argument>
-                                        <argument>org.apache.druid.extensions:druid-iceberg-extensions</argument>
```

Comment:
why are you removing this? This profile is bundle-contrib-exts which is supposed to package all contrib extensions in the bundle.

Reply:
Reverted this! Thanks for pointing it out.